app/controllers/admin/my_messages_controller.rb

```diff
@@ -0,0 +1,40 @@
+class Admin::MyMessagesController < AdminController
+  before_action :require_manager
+  before_action :find_user, only: [:index, :new, :create]
+
+  def index
+    @messages = @user.my_messages
+  end
+
+  def new
+    @message = @user.messages.new
+  end
+
+  def create
+    @message = @user.messages.new(message_params)
```

Comment:
not to use 'new', use 'create' instead,  so you don't have to pass user_ids
http://stackoverflow.com/questions/7297338/how-to-add-records-to-has-many-through-association-in-rails